apps/client/src/app/components/user-account-settings/user-account-settings.component.ts

```diff
@@ -69,12 +71,14 @@ export class UserAccountSettingsComponent implements OnDestroy, OnInit {
     private settingsStorageService: SettingsStorageService,
     private tokenStorageService: TokenStorageService,
     private userService: UserService,
+    private snackBar: MatSnackBar,
```

Comment:
Please move up to keep the alphabetical order ⬆️

Reply:
Didn't notice. Will update.